src/main/java/mate/academy/springboot/swagger/util/SortOrderUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortOrderUtil {
+    private SortOrderUtil() {
+    }
+
+    public static Sort getSorter(String[] sortBy) {
```

Comment:
don't make it static, create a field of type SortOrderUtil in controller and use this method where you need it